Pear/View/OnboardingSearchDropdownView.swift

```diff
@@ -159,11 +211,17 @@ extension OnboardingSearchDropdownView: UISearchBarDelegate, UITableViewDelegate
     /// Expands and updates search results table view when text is changed in the search bar.
     func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
         tableView.isHidden = false
-        resultsTableData = searchText.isEmpty ? [] : tableData.filter { $0.localizedCaseInsensitiveContains(searchText) }
+        switch searchType {
+        case .places:
+            searchPlaces(place: searchText)
```

Comment:
Should we [debounce](https://developer.apple.com/documentation/combine/publishers/share/debounce(for:scheduler:options:)) here? In other words, should we search every time text changes or only once text _stops_ changing for a certain amount of time? I'm worried about Google api rate limits. 

The link above is to a Combine framework function, so it won't help you directly. I linked it just for a description of the term debounce.